<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="EventDeliveryFailedException.xml" source-language="en-US" target-language="zh-TW">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac548b889544946a903dd75b269a8b9f2adb8827aff.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">48b889544946a903dd75b269a8b9f2adb8827aff</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>The exception that is thrown when an event that is raised from the host cannot be delivered to the workflow instance.</source>
          <target state="translated">當無法將從主應用程式引發的事件傳遞給工作流程執行個體時擲回的例外狀況。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>Typically the event is raised from an <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /&gt;</ph> on a workflow instance.</source>
          <target state="translated">一般而言，事件是從工作流程執行個體上的 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /&gt;</ph> 引發。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>This class cannot be inherited.</source>
          <target state="translated">這個類別無法被繼承。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>The following string is added to the event log when this exception is thrown: Event "{1}" on interface type "{0}" for instance id "{2}" cannot be delivered.</source>
          <target state="translated">當擲回這個例外狀況時，將下列字串加入至事件記錄：執行個體 ID "{2}" 的介面類型 "{0}" 上事件 "{1}" 無法傳遞。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>When using a state machine workflow, you might get an exception with the message "Queue '{0}' is not enabled."</source>
          <target state="translated">使用此狀態機器工作流程時，您可能會取得訊息為「未啟用佇列 '{0}'」的例外狀況。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>This happens when the current state of the state machine does not know how to handle a specific event.</source>
          <target state="translated">這種情況發生在狀態機器的目前狀態不知道如何處理特定事件時。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>For example, when some state other than the current state contains the <ph id="ph1">&lt;xref:System.Workflow.Activities.EventDrivenActivity&gt;</ph> that contains the <ph id="ph2">&lt;xref:System.Workflow.Activities.HandleExternalEventActivity&gt;</ph> that is represented by the queue '{0}'.</source>
          <target state="translated">例如，當目前狀態以外的某種狀態包含 <ph id="ph1">&lt;xref:System.Workflow.Activities.EventDrivenActivity&gt;</ph>，而且其中包含由佇列 '{0}' 所代表的 <ph id="ph2">&lt;xref:System.Workflow.Activities.HandleExternalEventActivity&gt;</ph> 時。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class.</source>
          <target state="translated">初始化 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String)">
          <source>An error message that explains the reason for the exception.</source>
          <target state="translated">解釋發生例外狀況原因的錯誤訊息。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class by using the error message that explains the reason for the exception.</source>
          <target state="translated">使用解釋例外狀況原因的錯誤訊息，初始化 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>An error message that explains the reason for the exception.</source>
          <target state="translated">解釋發生例外狀況原因的錯誤訊息。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that caused this exception.</source>
          <target state="translated">導致這個例外狀況的 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class by using the error message that explains the reason for the exception and the <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph> that caused this exception.</source>
          <target state="translated">使用解釋例外狀況原因的錯誤訊息和導致此例外狀況的 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph>，初始化 <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph> 類別的新執行個體。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>